css-display-3/display-contents-before-after-001-ref.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Reftest Reference</title>
+<link rel="author" title="Rune Lillesveen" href="mailto:rune@opera.com">
+<p>You should see the word PASS below.</p>
+PASS
```

Comment:
I've re-organized a bit to do this. I split the margin-collapsing through display:contents into a separate test (block-002)